src/__test__/actions/users.action.test.js

```diff
@@ -88,4 +92,20 @@ describe('Asset Action tests', () => {
       });
     });
   });
+
+  it('should dispatch CREATE_SECURITY_USER_SUCCESS when addSecurityUser is called successfully', () => {
+    mock.onPost(url3).reply(201, SecurityUser);
+    return store.dispatch(addSecurityUser()).then(() => {
+      expect(store.getActions()[0].type).toEqual(CREATE_SECURITY_USER_SUCCESS);
```

Comment:
Please use [jest's toContainEqual](https://jestjs.io/docs/en/expect#tocontainequalitem) instead of keeping track of the action index. Example:
```
expect(store.getActions()).toContainEqual({
  type: CREATE_SECURITY_USER_SUCCESS,
  payload: <whatever_is_the_mocked_payload>
});
```
Here is a usage example in [assetMakes actions' test](https://github.com/AndelaOSP/art-dashboard/blob/develop/src/__test__/actions/assetMakes.actions.test.js#L49-L52)